src/components/organisms/AppRouter/AdminSubrouter.tsx

```diff
@@ -5,31 +5,100 @@ import Admin from "pages/Admin/Admin";
 import { RoomsForm } from "pages/Admin/Venue/Rooms/RoomsForm";
 import { VenueWizard } from "pages/Admin/Venue/VenueWizard";
 
-import Admin_v2 from "pages/Admin/Admin_v2";
-import VenueWizard_v2 from "pages/Admin/Venue/VenueWizard/VenueWizard";
+import AdminV2 from "pages/Admin/Admin_v2";
+import VenueWizardV2 from "pages/Admin/Venue/VenueWizard/VenueWizard";
 import { AdminAdvancedSettings } from "pages/AdminAdvancedSettings";
 
 import { AdminVenueView } from "components/organisms/AdminVenueView";
+import { Provided } from "components/organisms/AppRouter/Provided";
```

Comment:
Since we're already in `components/organisms/AppRouter` this can just be a `./` import
```suggestion

import { Provided } from "./Provided";
```